script/bench_reading_events.rb

```diff
@@ -29,11 +29,11 @@ def create_events_schema(pg_connection)
 
 event_store = EventSourcery::Postgres.config.event_store
 
-EVENT_TYPES = %i[
+EVENT_TYPES = %i(
   item_added
   item_removed
   item_starred
-]
+).freeze
```

Comment:
I think `[]` is more readable than `()` as it's consistent with standard array literals. It's also what Rubocop asks for.